gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py

```diff
@@ -0,0 +1,315 @@
+import mock
+import unittest
+import subprocess
+
+from pecan import rest
+from pecan import set_config
+from pecan.testing import load_test_app
+
+mock_request = mock.Mock()
+
+from gbpservice.neutron.nfp.configurator.api.v1.controllers import controller
+
+
+class ControllerTestCase(unittest.TestCase, rest.RestController):
+
+    def setUp(self):
+        self.app = load_test_app('/home/hitesh/Desktop/\
```

Comment:
Why is this path static? This point to your Desktop path. How will this work on a different setup? Is this code tested on the Devstack setup provided?